test/performance/long-running-tests/README.md

```diff
@@ -0,0 +1,174 @@
+## Workload Benchmark for KubeStellar
+
+To generate the sample workload for KubeStellar performance experiments proceed as following:
+
+### Generate the workload traffic
+
+1. Clone the following clusterloader2 repo: 
+
+   ```bash 
+   git clone -b release-1.31 https://github.com/kubernetes/perf-tests.git
+   ```
+
+   Set the variable `CL2_DIR` to the path of the subdirectory `clusterloader2/` of the cloned repo. For example: 
+
+   ```bash 
+   export CL2_DIR=$HOME/perf-tests/clusterloader2
+   ```
+
+2. Configure clusterloader2 to generate KS performance workloads:
+
+   Starting from a local directory containing this KubeStellar git repo, run the following script based on your cluster environment:
+
+   a) cd into the `test/performance/common` directory from your local copy of the KubeStellar repo, for example:
+
+   ```bash 
+   cd .$HOME/kubestellar/test/performance/common
+   ```
+
+   b) set-up your environment:
+
+   ```bash 
+   ./setup-clusterloader2.sh --exp l
+   ```
+
+3. Configure the parameters of your workload:  
+
+   a) cd into the load configuration directory
+
+   ```bash 
+   cd  $CL2_DIR/testing/load/
+   ```
+  
+   b) configure the parameters to create the workload traffic (e.g., RandomizedLoad, SteppedLoad, Sequence, etc.)
+   
+   ```bash 
+   vi long-duration-exp-config.yaml
+   ``` 
+
+   More specifically, configure the following parameter: 
+
+   - numberOfWorkloadObjects: total number of objects to be created in the experiment `perf-test` namespace (*default value: "10"*)
```

Comment:
I see that it is already set to 10. The reader wonders what she is expected to change this to.

Reply:
The configuration of the input paramater `numberOfWorkloadObjects` is experiment specific. Thus, the reader is expected to decide beforehand how many objects he wants to be created in his experiment based on his experiment set-up or resources available to him.

Reply:
Additionally, I just added more explanation and provided an new example of how to configure the parameter `numberOfWorkloadObjects`.